### PR TITLE
added task_7

### PR DESCRIPTION
**Pull request by katy-lix**

### CODE REVIEW

File: task_4/Task 1/index1.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Task 1</title>
+</head>
+<body>
+    <div style="border-style:solid;padding:2px; ">
+      <div style=" background-color:green;margin:20px;height:30px;line-height:30px;color: honeydew;">
```
Comment (Killerkip10) [inline task_4/Task 1/index1.html]: Will be better to use padding instead of height and line-height

File: task_4/Task 2/index2.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+        <div style="border-style:solid;padding:2px;">
+            <div style="padding:10px;">
```
Comment (Killerkip10) [inline task_4/Task 2/index2.html]: This div is redundant. Remove it. And set this padding to parent div.

File: task_4/Task 3/task3.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>   
+   <div>
+      <div style=" padding-left:8px; text-align: left;background-color:rgb(245, 11, 30);margin:20px;height:40px;line-height:40px;color: honeydew;">
```
Comment (Killerkip10) [inline task_4/Task 3/task3.html]: Will be better to use padding instead of height and line-height

**Review by Killerkip10 — COMMENTED** [review]
## Homework 4
### Task 1
1.1 You didn't use margin (-5 points);

10 points
### Task 2
20 points
### Task 3
15 points
### Task 4
- Mismatch with layout. 'Trainings' logo is too small. (-5 points);
- 1.2 Background image should occupy all window height. Text inside should be centered vertically (-10 points);
- Mismatch with layout. Horizontal line below 'trainings' doesn't have margin. (-5 points);

30 points

**Total 75 points**

## Homework 5
### Task 1
- Mismatch with layout. Text in first line is not bold. (-5 points);
- Mismatch with layout. Text is not centered vertically. (-5 points);
- HTML Validator errors. (-5 points);

10 points
### Task 2
- Mismatch with layout. First line and other lines have different text color. (-5. points);
- The same Validator errors;

20 points
### Task 3
- Mismatch with layout. Table should has background-color: white (-5 points);
- Mismatch with layout. All page blocks should have the same margin-left and margin-right. In your case block with text and block with table have different margin-left/right. (-5 points);
- Mismatch with layout. In my screen buttons 'Записаться' and 'Программа курса' are located in one line. (-5 points);
- Mismatch with layout. Your table cels don't have padding. (-5 points);

30 points

**Total 60 points**

Please resolve all conversations.
**Deadline 04.08.2020 00:00 else -20 points**

File: task_5/task1.html [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+}
+table, td, th {
+  border: 1px solid black;
+  height: 25px;  
+}
+ .mir {width: 150px;
+ }
+ td {
+     
+     padding-bottom: 6px;
```
Comment (Killerkip10) [inline task_5/task1.html]: Why?)
Remove it

File: task_5/task1.html [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    <title>Document</title>
+</head>
+<style>
+table {border-collapse: collapse;
+       width: 100%; 
+       table-layout: auto ;
+}
+table, td, th {
+  border: 1px solid black;
+  height: 25px;
```
Comment (Killerkip10) [inline task_5/task1.html]: Currently your table doesn't have height: 25px;
Remove it.

File: task_5/task1.html [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+     padding-bottom: 6px;
+     text-align: left;
+     padding-left: 7px;
+ }
+ .cen {
+     text-align: center;
+  }
+</style>
+<body>
+    <p style="text-align: center;"><strong>Добыча драгоценных камней</strong></p>
```
Comment (Killerkip10) [inline task_5/task1.html]: Please use css property instead of `<strong>`

File: task_5/task1.html [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+<style>
+table {border-collapse: collapse;
+       width: 100%; 
+       table-layout: auto ;
+}
+table, td, th {
+  border: 1px solid black;
+  height: 25px;  
+}
+ .mir {width: 150px;
```
Comment (Killerkip10) [inline task_5/task1.html]: What is mir?)
Your class names are not informative.

File: task_6/index.html.txt [hunk 8]
```diff
@@ -0,0 +1 @@
+Blablabla
```
Comment (Killerkip10) [inline task_6/index.html.txt]: Please remove this file

File: task_5/Task2.html [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<style>
+    table {
+  border-collapse: collapse;
+  border: 1px solid rgb(42, 106, 201);
```
Comment (Killerkip10) [inline task_5/Task2.html]: Please use hex color format

File: task_5/Task2.html [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+  width: 100%;
+  color: rgb(22, 150, 236);
+  table-layout: auto;
+  text-align: center;
+  padding-bottom: 3px;
```
Comment (Killerkip10) [inline task_5/Task2.html]: useless

**Comment by Killerkip10** [comment]
Please align your  all code.

**Comment by Killerkip10** [comment]
Create task_7 folder and move the following files
Block/1_option.html 
Block/2_option.html 
Inline/1_option.html 
Inline/2_option.html

File: site/index5.html [hunk 11]
```diff
@@ -0,0 +1,138 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Katy</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link href='https://fonts.google.com/specimen/Roboto' rel='stylesheet' type='text/css'>
+</head>
+<body>
+<header  >
+    <div style=" background-color:rgb(73, 68, 68,01) ; height:40px; line-height: 30px;letter-spacing:1px; word-spacing:10px;padding-left: 10px; width: 100%;margin: 0;font-size: 100%;">
```
Comment (Killerkip10) [inline site/index5.html]: I think this `<div>` is redundant. You can remove it and set all this css properties to the `<header>`

File: site/index5.html [hunk 12]
```diff
@@ -0,0 +1,138 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Katy</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link href='https://fonts.google.com/specimen/Roboto' rel='stylesheet' type='text/css'>
+</head>
+<body>
+<header  >
+    <div style=" background-color:rgb(73, 68, 68,01) ; height:40px; line-height: 30px;letter-spacing:1px; word-spacing:10px;padding-left: 10px; width: 100%;margin: 0;font-size: 100%;"> 
+      <span ><a href ="https://www.ebay.com" target="blank" style="text-decoration: none; color: rgb(250, 247, 237) ;word-spacing:15px;" >Trainings<sup>&reg;</sup> </a> </span>
```
Comment (Killerkip10) [inline site/index5.html]: remove <span>

File: site/index5.html [hunk 13]
```diff
@@ -0,0 +1,138 @@
...
+       <span><a href ="https://www.ebay.com" target="blank" style="text-decoration: none; color: rgb(250, 247, 237) ; " >List </a> </span>
+      <span><a href ="https://www.ebay.com" target="blank" style="text-decoration: none; color: rgb(250, 247, 237); " >Blog </a> </span>
+      <span><a href ="https://www.ebay.com" target="blank" style="text-decoration: none; color: rgb(250, 247, 237) ;" >FAQ </a> </span>
+    </div>
+ </header>
+
+    <div style="display: flex;" >
```
Comment (Killerkip10) [inline site/index5.html]: Why did you use it?)

File: site/index5.html [hunk 14]
```diff
@@ -0,0 +1,138 @@
...
+        <div  style="  color: rgb(250, 250, 250);height:520px; padding-top: 80px;padding-left: 60px;font-size: 300%;">
```
Comment (Killerkip10) [inline site/index5.html]: Why did you use `%` for `font-size`? Just use `px`

File: site/index5.html [hunk 15]
```diff
@@ -0,0 +1,138 @@
...
+ </header>
+
+    <div style="display: flex;" >
+        <div  style="  color: rgb(250, 250, 250);height:520px; padding-top: 80px;padding-left: 60px;font-size: 300%;"> 
+             <div>Start your</div>
+            <div>career</div>
+       </div>
+    </div>
+    <main >
+     <div style="background-color: rgb(224, 223, 220); width: 100%;margin-left: 5px;margin-left: 0px;">
```
Comment (Killerkip10) [inline site/index5.html]: margin-left: 5px; and then margin-left: 0px; ?)))

Comment (Killerkip10) [inline site/index5.html]: <div> already has width: 100% by default. Remove it

File: site/index5.html [hunk 16]
```diff
@@ -0,0 +1,138 @@
...
+         <div style=" height:40px; line-height:50px; background-color: rgb(224, 223, 220);font-family: Roboto, sans-serif; color:black;  text-align: center; ">
```
Comment (Killerkip10) [inline site/index5.html]: You have already set font-family in .css file.

Comment (Killerkip10) [inline site/index5.html]: Remove height, line-height, background-color

File: site/index5.html [hunk 17]
```diff
@@ -0,0 +1,138 @@
...
+            <hr>
+       
+            
+            <div style=" background-color: beige;margin: 5px; "> 
+
+                    <table style=" background-color: beige; width: 100%; ">
+
+                        <tr style="height: 35%;line-height: 20px;" >
+                              <td style="width: 40%; "><span class="go">Програм&shy;мирование на GO</span> 
+                                  <div style="font-size: 80%;">20.08.2020-20.12.2020</div>
```
Comment (Killerkip10) [inline site/index5.html]: Don't use `%` for font-size. It is not your case. Just `px`

File: site/index5.html [hunk 18]
```diff
@@ -0,0 +1,138 @@
...
+
+            <div style="margin: 10px;text-align: center;font-family: Roboto, sans-serif;"> <strong>UPCOMING COURSES</strong> </div>
+            <hr>
+       
+            
+            <div style=" background-color: beige;margin: 5px; "> 
+
+                    <table style=" background-color: beige; width: 100%; ">
+
+                        <tr style="height: 35%;line-height: 20px;" >
```
Comment (Killerkip10) [inline site/index5.html]: Why did you use height?)
I think it is useless

Comment (Killerkip10) [inline site/index5.html]: Remove line-height

File: site/index5.html [hunk 19]
```diff
@@ -0,0 +1,138 @@
...
+                              <td style="width: 40%; "><span class="go">Програм&shy;мирование на GO</span>
```
Comment (Killerkip10) [inline site/index5.html]: Remove width. Teble cels will be stretched by width automatically.

File: site/style.css [hunk 20]
```diff
@@ -0,0 +1,96 @@
+
+body {
+    font-family: Roboto, sans-serif;
+    background-image:url(pexels-photo-583846.jpeg);
```
Comment (Killerkip10) [inline site/style.css]: Why did you set it for body???

File: site/style.css [hunk 21]
```diff
@@ -0,0 +1,96 @@
...
+}
+
+.green {
+    background: #5ece3c;
+    border-bottom: solid 3px #7d9b43;
+    color: #ffffff;
+    text-shadow: 0 1px 0 #d0e5a4; 
+    padding: 8px 40px;
+    border-radius: 5px;
+    font-size:100%;
```
Comment (Killerkip10) [inline site/style.css]: useless

File: site/style.css [hunk 22]
```diff
@@ -0,0 +1,96 @@
...
+    line-height: 30px;
```
Comment (Killerkip10) [inline site/style.css]: Remove. You already have padding

File: site/style.css [hunk 23]
```diff
@@ -0,0 +1,96 @@
...
+    border-radius: 5px;
+    font-size:100%;
+    line-height: 30px;       
+    text-align: center;
+    text-decoration: none;
+    
+    }
+    
+.green:hover {
+background: #7d9b43;
```
Comment (Killerkip10) [inline site/style.css]: In this block you should change only background-color!! Remove other not changed properties.

File: site/style.css [hunk 24]
```diff
@@ -0,0 +1,96 @@
...
+body {
+    font-family: Roboto, sans-serif;
+    background-image:url(pexels-photo-583846.jpeg);
+    background-repeat:no-repeat;
+    background-size:cover;
+    background-attachment:fixed;
+    margin:0px;
+}
+
+.green {
```
Comment (Killerkip10) [inline site/style.css]: What is green?)
Class names is not informative

File: site/style.css [hunk 25]
```diff
@@ -0,0 +1,96 @@
...
+
+}
+.dfloat:hover{
+    display: flex;
+    flex-wrap: wrap;
+    float: left;
+    width:22%;
+    min-width: 300px;
+    padding:7px;
+    background-color: rgb(253, 253, 253);
```
Comment (Killerkip10) [inline site/style.css]: Also, you change only background-color. Remove other properties in this block

File: task_4/Task 1/index1.html (hunk 1 above)
Comment (katy-lix) [reply task_4/Task 1/index1.html]: done

File: site/index5.html [hunk 26]
```diff
@@ -0,0 +1,142 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Katy</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link href='https://fonts.google.com/specimen/Roboto' rel='stylesheet' type='text/css'>
+</head>
+<body>
+    <div class=" main">
+<header style="text-decoration: none; background-color:rgb(73, 68, 68,01) ; height:40px; line-height: 30px;color: rgb(255, 255, 255) ;letter-spacing:1px; word-spacing:10px;padding-left: 10px;font-size: 100%;" >
```
Comment (katy-lix) [inline site/index5.html]: fixed

File: site/style.css (hunk 20 above)
Comment (katy-lix) [reply site/style.css]: fixed

**Pull request closed without merging**